modules/integration/src/test/java/io/fluo/integration/impl/WeakNotificationOverlapIT.java

```diff
@@ -128,7 +127,7 @@ public void testOverlap() throws Exception {
     ttx7.mutate().row(1).col(ntfyCol).weaklyNotify();
     ttx7.done();
 
-    Assert.assertEquals(2, countNotifications());
+    Assert.assertEquals(1, countNotifications());
 
     new TotalObserver().process(ttx6, Bytes.of("1"), ntfyCol);
     // should not delete notification created by ttx7
```

Comment:
the comment is still good.  there are two notifications in Accumulo, however only one is shown now.    The delete should go between the two.